tools/project_manager/manage_image_sources.php

```diff
@@ -152,7 +152,7 @@ class ImageSource
     public int $ok_keep_images; // TODO: bool
     public int $ok_show_images; // TODO: bool
     public string $public_comment;
-    public string $internal_comment;
+    public ?string $internal_comment;
```

Comment:
`text` fields can't have default values so this ensures `null` is a valid entry from the DB, despite this form always setting it to `''` on creation.